Dockerfile

```diff
@@ -20,5 +20,9 @@ RUN pip install --no-cache-dir --upgrade -r /code/requirements.txt
 
 COPY . /code/
 
-# Heroku uses PORT, Azure App Services uses WEBSITES_PORT, Fly.io uses 8080 by default
-CMD ["sh", "-c", "uvicorn server.main:app --host 0.0.0.0 --port ${PORT:-${WEBSITES_PORT:-8080}}"]
+ARG RENDER_EXTERNAL_HOSTNAME
```

Comment:
Might be good to add a comment above the Render/weaviate section

Reply:
How does it look with the new changes?

Reply:
That's not a TODO, but a comment explaining what the next two lines are doing. I can make that more clear.